### DISCUSSION TITLE
Trigger manual search

### DISCUSSION THREAD

**Discussion by fredleput** [KevinVandy/material-react-table#474] [Q&A]
Hi,
I wonder how to trigger a manual search. My goal is :

- step 1 : user perform a search
- step 2 : user edit a row and saves modification (form values are post to my API)
- step 3 : i reload the data to get them updated
- How can i set the searched value and filter the table accordingly to the search performed in step 1 ? So that the user immediately see the change.

Thx,

Fred.

**Comment by KevinVandy** [comment]
You can put this is the initialState
